README.md

```diff
@@ -46,11 +46,23 @@ Node config:
 
 ## Installation
 
-```
+```bash
 yarn add --dev eslint @callstack/eslint-config
 ```
 
-*Note: We're using `yarn` to install deps. Feel free to change commands to use `npm` 3+ and `npx` if you like*
+*Note: We're using `yarn` to install dependencies. Feel free to change commands to use `npm` 3+ and `npx` if you like*
+
+### For TypeScript project
+
+In order to use this config in TypeScript project make sure you have installed following dependencies:
+
+* [`@typescript-eslint/eslint-plugin`](https://yarnpkg.com/en/package/@typescript-eslint/eslint-plugin)
+* [`@typescript-eslint/parser`](https://yarnpkg.com/en/package/@typescript-eslint/parser)
+* [`typescript`](https://yarnpkg.com/en/package/typescript)
+
+```bash
+yarn add --dev @typescript-eslint/eslint-plugin @typescript-eslint/parser typescript
```

Comment:
We can add `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` as deps by default so user doesn't need to install these. Wdyt @thymikee?

Reply:
I'm ok with that, cc @zamotany

Reply:
Let's proceed and add these to dependencies